src/ExpressMiddlewares.js

```diff
@@ -123,7 +122,7 @@ class ExpressMiddlewares {
             " \"request_method\": \":method\", \"uri\": \":uri\", \"query_string\": \":query_string\"," +
             " \"response_time\": \":response-time\", \"protocol\": \":protocol\", \"server_name\": \":server_name\"," +
             " \"current_color\": \":service_color\", \"remote_client_id\": \":remote_client_id\", " +
-            optKeysString + " \"bytes_received\": \":bytes_received\" }",
+            `${optKeys.length ? optKeys.join() + ',' : optKeys.join()}` + " \"bytes_received\": \":bytes_received\" }",
```

Comment:
if space is added at the top (2n comment) you dont need the space here before \"bytes_received\

Reply:
For my proposal above, this space would be necessary. But anyway let me know how you think.